core/serialization/blocks.js

```diff
@@ -424,4 +448,22 @@ const loadConnection = function(connection, connectionState) {
         connection);
   }
 };
-exports.load = load;
+
+// TODO (#5146): Remove this from the serialization system.
```

Comment:
I think you need to remove the space after `TODO` in order to take advantage of automatic linking and the like